eva-server/src/main/java/uk/ac/ebi/eva/server/ws/ga4gh/beaconv2/BeaconServiceV2.java

```diff
@@ -161,11 +162,18 @@ private void checkParameters(String chromosome, String referenceBases, Long star
             throw new IllegalArgumentException("Please provide reference bases");
         }
 
-        if (start != null && start < 0) {
+        if (start == null) {
+            if (startMin == null || startMax == null) {
+                throw new IllegalArgumentException("Please provied either start or both startMin and startMax");
+            }
+        } else if (start < 0) {
             throw new IllegalArgumentException("Please provide a positive start number");
         }
-
-        if (end != null && end < 0) {
+        if (end == null) {
+            if (endMin == null || endMax == null) {
+                throw new IllegalArgumentException("Please provied either end or both endMin and endMax");
```

Comment:
> the use of "start" without an "end" parameter requires the use of "referenceBases"

but it's allowed not to provide any end* parameter

Reply:
I have removed this check for end parameter :)